xml/installation-installation-rhel-install_rhel_compute.xml

```diff
@@ -2,15 +2,15 @@
 <!DOCTYPE chapter [
  <!ENTITY % entities SYSTEM "entities.ent"> %entities;
 ]>
-<chapter xml:id="install_rhel_compute"
+<chapter vendor="hpe" xml:id="install_rhel_compute"
  xmlns="http://docbook.org/ns/docbook" version="5.1"
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink">
  <title>Installing &rhla; Compute</title>
 
- <xi:include href="installation-installation-rhel-rhel_overview.xml"/>
- <xi:include href="installation-installation-rhel-rhel_support.xml"/>
- <xi:include href="installation-installation-rhel-install_rhel.xml"/>
- <xi:include href="installation-installation-rhel-provisioning_rhel.xml"/>
- <xi:include href="installation-installation-rhel-install_rhel_ceph.xml"/>
+ <xi:include vendor="hpe" href="installation-installation-rhel-rhel_overview.xml"/>
```

Comment:
There is no need to do this (and many of the other things done here), this file is already excluded here: https://github.com/SUSE-Cloud/doc-cloud/blob/develop/xml/installation-installation-cloudinstallation_overview.xml#L108

To check which files are included for the Helion Installation Guide, do `daps -d DC-hpe-helion-installation list-srcfiles --xmlonly` (for the SOC Installation Guide, use `DC-suse-openstack-cloud-installation` of course). You can also compare the file lists. E.g. to show the files that are only included in the Helion Installation Guide:

```
# comm -2 -3 \
  <(daps -d DC-hpe-helion-installation list-srcfiles --xmlonly | tr ' ' '\n' | sort -u) \
  <(daps -d DC-suse-openstack-cloud-installation list-srcfiles --xmlonly | tr ' ' '\n' | sort -u)

[...]/doc-cloud/xml/installation-installation-rhel-install_rhel_ceph.xml
[...]/doc-cloud/xml/installation-installation-rhel-install_rhel_compute.xml
[...]/doc-cloud/xml/installation-installation-rhel-install_rhel.xml
[...]/doc-cloud/xml/installation-installation-rhel-provisioning_rhel.xml
[...]/doc-cloud/xml/installation-installation-rhel-rhel_overview.xml
[...]/doc-cloud/xml/installation-installation-rhel-rhel_preinstall.xml
[...]/doc-cloud/xml/installation-installation-rhel-rhel_support.xml

```